client/lib/domains/index.js

```diff
@@ -2,9 +2,12 @@
  * External dependencies
  */
 import inherits from 'inherits';
-import some from 'lodash/some';
-import includes from 'lodash/includes';
-import find from 'lodash/find';
+import {
+	endsWith,
+	find,
+	includes,
+	some
+} from 'lodash';
```

Comment:
We do direct imports :) Try grepping our source code for `} from 'lodash'` - that's how I stumbled upon it.
Alarmed by your comment I dug deeper (worried that we indeed might be pulling in lodash as a whole), but it turns out that thanks to the magic of [babel-plugin-loadsh](https://github.com/lodash/babel-plugin-lodash) (enabled [here](https://github.com/Automattic/wp-calypso/blob/master/.babelrc#L14)), these import statements are transformed to cherry-pick only the needed lodash modules. So all is good 👍 Good catch though :)